pkg/crds/crd.go

```diff
@@ -68,49 +75,56 @@ func List() []crd.CRD {
 }
 
 func customizeBackup(backup *apiext.CustomResourceDefinition) {
-	properties := backup.Spec.Validation.OpenAPIV3Schema.Properties
-	spec := properties["spec"]
-	spec.Required = []string{"resourceSetName"}
-	resourceSetName := spec.Properties["resourceSetName"]
-	resourceSetName.Description = "Name of the ResourceSet CR to use for backup"
-	spec.Properties["resourceSetName"] = resourceSetName
-	encryptionConfig := spec.Properties["encryptionConfigSecretName"]
-	encryptionConfig.Description = "Name of the Secret containing the encryption config"
-	spec.Properties["encryptionConfigSecretName"] = encryptionConfig
-	schedule := spec.Properties["schedule"]
-	schedule.Description = "Cron schedule for recurring backups"
-	examples := make(map[string]interface{})
-	examples["Standard crontab specs"] = "0 0 * * *"
-	examples["Descriptors"] = "@midnight"
-	byteArr, err := json.Marshal(examples)
-	if err == nil {
-		schedule.Example = &apiext.JSON{Raw: byteArr}
+	for _, version := range backup.Spec.Versions {
+		properties := version.Schema.OpenAPIV3Schema.Properties
```

Comment:
hmm so are we doing in a loop and only take the last one?

Reply:
The `properties` is reset to be the current version’s during each iteration, so the changes are applied to each version. is it right?

Reply:
I see the point.